boards/shields/ssd1306/sh1106_128x64.overlay

```diff
@@ -1,29 +1,11 @@
 /*
  * Copyright (c) 2019 PHYTEC Messtechnik GmbH
+ * Copyright (c) 2022 Miguel Dardenne <miguel.dardenne@gmail.com>
  *
  * SPDX-License-Identifier: Apache-2.0
  */
 
-/ {
-	chosen {
-		zephyr,display = &sh1106_sh1106_128x64;
-	};
-};
-
-&arduino_i2c {
-	status = "okay";
-
-	sh1106_sh1106_128x64: ssd1306@3c {
-		compatible = "solomon,ssd1306fb";
-		reg = <0x3c>;
-		width = <128>;
-		height = <64>;
-		segment-offset = <2>;
-		page-offset = <0>;
-		display-offset = <0>;
-		multiplex-ratio = <63>;
-		segment-remap;
-		com-invdir;
-		prechargep = <0x22>;
-	};
-};
+/*
```

Comment:
That is problematic, you break sh1106_128x64, ssd1306_128x64, and ssd1306_128x32 shields, there is also ssd1306_128x64_spi. I suggest to not introduce ssd1306_*_arduino shields.